tests/requirements.txt

```diff
@@ -23,7 +23,7 @@ pytest-flask==0.10.0
 pytest==3.0.3
 pytest-cov==2.5.1
 python-dateutil==2.5.3    # via freezegun
-requests==2.11.1
+requests>=2.20.0
```

Comment:
@SriKrishnaPaparaju requirements.txt will need a resolved version. So, please replace `>=` with `==`. Also, there is flask dependency in this file which has to be updated with the version `==1.0.0`.